arroyo/dead_letter_queue/policies/count.py

```diff
@@ -0,0 +1,18 @@
+from arroyo.dead_letter_queue.policies.abstract import (
+    DeadLetterQueuePolicy,
+    InvalidMessage,
+)
+from arroyo.types import Message, TPayload
+
+
+class CountInvalidMessagePolicy(DeadLetterQueuePolicy[TPayload]):
+    def __init__(self, limit: int) -> None:
+        self.__limit = limit
+        self.__count = 0
+
+    def handle_invalid_message(
+        self, message: Message[TPayload], e: InvalidMessage
+    ) -> None:
+        self.__count += 1
+        if self.__count > self.__limit:
```

Comment:
Does the count reset if the exception gets hit?

Reply:
For now, this policy just allows a certain number of `InvalidMessage`'s till it starts raising all incoming exceptions. If we reset the count, this policy changes to be more like `raise every nth exception` instead of `raise after n exceptions`. The only way to reset would be by restarting this DLQ? I'm not sure how to tell it to reset the count without potentially a runtime config or just changing the policy to resetting after an exception is hit.

Reply:
Revamped the entire policy to work as a limit per time unit.